src/core/public/chrome/ui/header/header_logo.tsx

```diff
@@ -98,44 +83,27 @@ function onClick(
   }
 }
 
-export const DEFAULT_LOGO = 'ui/logos/opensearch_dashboards.svg';
-export const DEFAULT_DARK_LOGO = 'ui/logos/opensearch_dashboards_darkmode.svg';
-
 interface Props {
   href: string;
   navLinks$: Observable<ChromeNavLink[]>;
   forceNavigation$: Observable<boolean>;
   navigateToApp: (appId: string) => void;
   branding: ChromeBranding;
-  basePath: HttpStart['basePath'];
+  logos: Logos;
   // What background is the logo appearing on; this is independent of theme:darkMode
   theme?: EuiHeaderProps['theme'];
 }
 
-export function HeaderLogo({
-  href,
-  navigateToApp,
-  branding,
-  basePath,
-  theme = 'default',
-  ...observables
-}: Props) {
+export function HeaderLogo({ href, navigateToApp, branding, logos, theme, ...observables }: Props) {
   const forceNavigation = useObservable(observables.forceNavigation$, false);
   const navLinks = useObservable(observables.navLinks$, []);
-  const {
-    logo: { defaultUrl: customLogoUrl, darkModeUrl: customDarkLogoUrl } = {},
-    applicationTitle = 'opensearch dashboards',
-  } = branding;
+  const { applicationTitle = 'opensearch dashboards' } = branding;
 
-  // Attempt to find a suitable custom branded logo before falling back on OSD's
-  let logoSrc = theme === 'dark' && customDarkLogoUrl ? customDarkLogoUrl : customLogoUrl;
-  let testSubj = 'customLogo';
-
-  // If no custom branded logo was set, use OSD's
-  if (!logoSrc) {
-    logoSrc = `${basePath.serverBasePath}/${theme === 'dark' ? DEFAULT_DARK_LOGO : DEFAULT_LOGO}`;
-    testSubj = 'defaultLogo';
-  }
+  const {
+    [theme === 'dark' ? 'dark' : 'light']: { url: logoURL },
```

Comment:
I still find the naming super confusing for this component prop - I hate that there's a "theme" that is unrelated to the actual application theme in use. I think I'd like a reminder comment that we need to conditionally set the logo based on the background of the header, which is set by this prop.

Reply:
`theme` is a naming inherited from OUI. When we move to color-scheme usage, all of these will be replaced with CSS as opposed to props.

Reply:
yeah, I meant that I don't like that OUI prop name, and thought a comment might help the reader remember which theme we're talking about here.

Reply:
Now i got what you meant; I have renamed the prop (since I created it for this change) to make that distinction.